packages/flutter_test/lib/src/finders.dart

```diff
@@ -178,6 +178,20 @@ class CommonFinders {
   Finder byElementPredicate(ElementPredicate predicate, { String description, bool skipOffstage: true }) {
     return new _ElementPredicateFinder(predicate, description: description, skipOffstage: skipOffstage);
   }
+
+  /// Looks for widgets that match the pattern of descendant finder under the
+  /// widget tree with ancestor as the root.
+  ///
+  /// Example:
+  ///
+  ///     expect(tester, hasWidget(find.descendant(
```

Comment:
So at some point we changed the API, but forgot to change the docs.

Could you adjust the docs for yours and all the others to say something more like:

```dart
expect(find.blabla, findsOneWidget);
```

...?